asset-transfer-basic/chaincode-javascript/.gitignore

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
This is the standard .gitignore file .. but most is redundant for our needs.. perhaps worth slimming down.

Reply:
done.